packages/browser/src/Session.ts

```diff
@@ -65,6 +65,17 @@ export interface IHandleIncomgingRedirectOptions {
    * {@see onSessionRestore}
    */
   restorePreviousSession?: boolean;
+
+  /**
+   * Inrupt's Enterprise Solid Server sets a cookie to allow the browser to
+   * access private resources on a Pod. In order to mitigate the logout-on-refresh
+   * issue on the short term, the server also implemented a session endpoint
+   * enabling the client app to know whether the cookie is set. If your app
+   * supports the newest session restore approach, or if you want to prevent
+   * your app to look up the session endpoint on your resource server, this
+   * option may be set to false. It defaults to true for backward compatibility.
```

Comment:
I'd think I'd only mention that you might want to set this to `false` if you want to prevent this lookup, and then explicitly call out separately that this value will always be overridden to `false` if `restorePreviousSession` is `true`. Since it can be confusing when an option doesn't have the stated effect.

Reply:
Clarified in b2f97b9b387a44bac4191ddb5586f86ed61bd549